frontend/packages/dev-console/src/components/pipelines/utils/pipeline-operator.ts

```diff
@@ -0,0 +1,27 @@
+import { compare, parse, SemVer } from 'semver';
+import {
+  ClusterServiceVersionKind,
+  ClusterServiceVersionModel,
+  ClusterServiceVersionPhase,
+} from '@console/operator-lifecycle-manager';
+import { k8sList } from '@console/internal/module/k8s';
+
+export const getPipelineOperatorVersion = async (namespace: string): Promise<SemVer | null> => {
+  const allCSVs: ClusterServiceVersionKind[] = await k8sList(ClusterServiceVersionModel, {
+    ns: namespace,
+  });
+  const matchingCSVs = allCSVs.filter(
+    (csv) =>
+      csv.metadata?.name?.startsWith('openshift-pipelines-operator') &&
+      csv.status?.phase === ClusterServiceVersionPhase.CSVPhaseSucceeded,
```

Comment:
@andrewballantyne @christianvogt Does it makes sense to check the status.phase of the Pipeline Operator to get the right version? If this is not required I could use `k8sListPartialMetadata` insteadof `k8sList` which makes the API response much smaller.